src/mantarray_waveform_analysis/peak_detection.py

```diff
@@ -76,13 +76,13 @@ def peak_detector(
         gmr_signal * peak_invertor_factor,
         width=minimum_required_samples_between_twitches / 2,
         distance=minimum_required_samples_between_twitches,
-        prominence=max_prominence / 22,
+        prominence=max_prominence / 2,
```

Comment:
did you have to go all the way down to 2 to get it to work?  I should have put some more comments/notes when I created the original `/ 22`, but it is biologically relevant to try and keep this number small.  Throughout a multi-minute recording, the size of the twitches could easily change by more than a factor of 2, so we don't want to force all peaks be a similar size as the largest one in the whole recording.

Ultimately what we may want to do is something like breaking up the whole recording into ~10 second overlapping chunks (overlapping so we don't lose the twitches at the edges of each 10 second chunk), run peak detection on those, and then reassemble everything into a composite of the twitches for the whole long recording.  It is biologically unlikely that the amplitude of the twitches would drop by more than 3X during a 10 second period

Reply:
for now, if there's a way to put it at `/ 3` or `/ 4` and it still works, then let's go with that. but if it does take `/ 2` to work on Travis' data then let's just leave it there for now